desc/compute/_omnigenity.py

```diff
@@ -390,6 +650,26 @@ def _B_modes(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="Boozer transform modes norm",
+    label="",
+    units="~",
+    units_long="None",
+    description="Inner product norm for boozer modes basis.",
```

Comment:
I know what this is, but it's not obvious from the name/description. Is this something that has to be its own compute quantity? It's really just a weighting used for other calculations, and it's a constant determined by the basis modes. I think it would make more sense to have it passed in with the `transforms` (or even add it as an attribute to the Basis class?).

Reply:
It is a constant which has to be re-calculated for a few other things, so to me makes sense to ahve it computed once and stored. Whether that is here or elsewhere does not really matter, but to pass it in transforms I would need to compute it in like get_transforms or something, and that will require other special logic as well. If possible I'd like to keep it as is here, if people are curious they can check the source code like the other stuff in our list of variables.

This also could be used to transform other quantities to Boozer as well, so it is useful to have as a compute quantiity I think.